azurerm/internal/services/storage/storage_account_resource.go

```diff
@@ -1851,6 +1992,44 @@ func flattenBlobPropertiesDeleteRetentionPolicy(input *storage.DeleteRetentionPo
 	return deleteRetentionPolicy
 }
 
+func flattenBlobPropertiesChangeFeed(input *storage.ChangeFeed) []interface{} {
+	changeFeed := make([]interface{}, 0)
+
+	if input == nil || input.Enabled == nil || !*input.Enabled {
+		return changeFeed
+	}
+
+	days := 0
+	if input.RetentionInDays != nil {
+		days = int(*input.RetentionInDays)
+	}
+
+	return []interface{}{map[string]interface{}{
+		"retention_in_days": days,
+	},
+	}
+}
+
+func flattenBlobPropertiesLastAccessTimeTrackingPolicy(input *storage.LastAccessTimeTrackingPolicy) []interface{} {
+	policy := make([]interface{}, 0)
+
+	if input == nil || input.Enable == nil || !*input.Enable {
+		return policy
+	}
+
+	days := 0
+	if input.TrackingGranularityInDays != nil {
+		days = int(*input.TrackingGranularityInDays)
+	}
+
+	return []interface{}{map[string]interface{}{
+		"name":                input.Name,
+		"granularity_in_days": days,
+		"blob_type":           utils.FlattenStringSlice(input.BlobType),
+	},
+	}
```

Comment:
_minor_ I'm surprised the linter didn't complain about this but:

```suggestion
	return []interface{}{
		map[string]interface{}{
			"name":                input.Name,
			"granularity_in_days": days,
			"blob_type":           utils.FlattenStringSlice(input.BlobType),
		},
	}
```